conf/nutch-default.xml

```diff
@@ -1896,4 +1896,33 @@ CAUTION: Set the parser.timeout to -1 or a bigger value than 30, when using this
   </description>
 </property>
 
+<!-- index-links plugin -->
```

Comment:
by best practice, the 3 properties would share a common prefix, so that it's clear to which plugin they belong